TST:Simple tests for find_last() and insert()

arkilic opened the pull request.

Commit:
TST:Tests for find_last() and insert()

tacaswell commented on metadatastore/test/test_commands.py:
```diff
@@ -504,11 +505,27 @@ def test_double_run_stop():
         mdsc.insert_run_stop(run_start_uid, ttime.time(),
                              uid=str(uuid.uuid4()))
 
+def prep_header():
```
Can you do this as a fixture?

Commit:
MNT:Fixture util func

tacaswell commented on metadatastore/test/test_commands.py:
```diff
@@ -503,12 +504,28 @@ def test_double_run_stop():
     last, = mdsc.find_last()
 
 
+def test_find_last(prep_header):
```
These should also take in the `mds_all` fixture so that these tests run against all three of the mds versions.

tacaswell replied:
Almost all of these are already duplicated in `test_mds` (just not removed here).

tacaswell committed:
API: remove v0 compatibility layer

If you still need to access an old database (none of which should exist
on the floor) use an old version of mds to migrate to v1.

Commit:
TST:Deprecate v0 tests

Commit:
MNT:Merge with upstream that deprecates v0
